astropy/wcs/utils.py

```diff
@@ -163,6 +239,30 @@ def _celestial_frame_to_wcs_builtin(frame, projection="TAN"):
         ycoord = "TLAT"
         wcs.wcs.radesys = "ITRS"
         wcs.wcs.dateobs = frame.obstime.utc.isot
+        if issubclass(frame.representation_type, BaseGeodeticRepresentation):
+            wcs.wcs.name = "Geodetic Terrestrial Representation"
```

Comment:
Is it tested that these round-trip? It would be nice to have it for sure! (Note I'm also OK with just not touching this part, but if we add anything we should test it...)

Reply:
As you are OK with just not touching them and I feel uncomfortable in adding ad hoc wcs descriptions about reference systems out of the scope of this PR, I'd rather remove this lines and the following.